cmd/snap-confine/snap-confine.c

```diff
@@ -162,12 +175,9 @@ int main(int argc, char **argv)
 			debug
 			    ("resetting PATH to values in sync with core snap");
 			setenv("PATH",
-			       "/usr/local/sbin:"
```

Comment:
Interesting that `indent` would change this. Can you try to not alter this part?